src/test/java/co/rsk/bitcoinj/script/RedeemScriptValidatorTest.java

```diff
@@ -384,4 +384,102 @@ public void isOpnN_invalid_opcode() {
         ScriptChunk chunk = new ScriptChunk(ScriptOpCodes.OP_DROP, null);
         Assert.assertFalse(RedeemScriptValidator.isOpN(chunk));
     }
+
+    @Test
+    public void hasP2shErpRedeemScriptStructure_whenEmptyScript_shouldReturnFalse() {
+        Script emptyScript = new Script(new byte[]{});
+
+        boolean actualHasP2shErpRedeemScriptStructure = RedeemScriptValidator.hasP2shErpRedeemScriptStructure(
+            emptyScript.getChunks());
+
+        Assert.assertFalse(actualHasP2shErpRedeemScriptStructure);
+    }
+
+    @Test
+    public void hasP2shErpRedeemScriptStructure_whenScriptSig_shouldReturnFalse() {
+        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(defaultRedeemScriptKeys);
```

Comment:
wdyt about defining this as a global variable? its being used in a bunch of places

Reply:
and creating an `assertScriptDoesNotHaveP2shErpStructure` method like this:

```
assertScriptDoesNotHaveP2shErpStructure(Script script) {
        List<ScriptChunk> scriptChunks = script.getChunks();
        assertFalse(hasP2shErpRedeemScriptStructure(scriptChunks));
}
```

to recreate parameterized tests manually. wdyt?

Reply:
Not possible in bitcoinj-thin for now.

Reply:
I don't understand the first comment. Do you mean having this `private Script redeemScript` at class level? Then just reassigning the value in each test?

Reply:
not reassigning right? you are defining the same one in a couple of tests, with the `defaultRedeemScriptKeys`

Reply:
updated